common/Linux/LnxMisc.cpp

```diff
@@ -69,8 +72,8 @@ std::string GetOSVersionString()
 #endif
 }
 
+#ifndef DBUS_API
 #ifdef X11_API
```

Comment:
```suggestion
#if !defined(DBUS_API) && defined(X11_API)
```

and drop the #endif at the end